package.json

```
@@ -17,6 +17,7 @@
     "start:dynamic": "lerna run dev --scope dynamic-pages --stream",
     "start:common": "lerna run start --scope common --stream",
     "start:home": "lerna run start --scope homepage --stream",
+    "start:storybook": "lerna run storybook --scope common --stream",
```

```suggestion
    "start:storybook": "lerna run storybook --scope @codesandbox/common --stream",
```

Lerna expects the package name